spec/requests/splash_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Splashes', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/splash/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] This test appears to be failing. Kindly fix it.

  ![image](https://user-images.githubusercontent.com/36542986/235440625-5c2ad13d-4e3e-4904-962b-6c97922af8cf.png)